src/sdk/VoiceProfileClient.ts

```diff
@@ -80,6 +81,62 @@ export class VoiceProfileClient {
         this.implClientSetup();
     }
 
+    /**
+     * Get current information of a voice profile
+     * @member VoiceProfileClient.prototype.retrieveEnrollmentResultAsync
+     * @function
+     * @public
+     * @param {VoiceProfile} profile Voice Profile to retrieve info for
+     * @param cb - Callback invoked once Voice Profile has been created.
+     * @param err - Callback invoked in case of an error.
+     */
+    public retrieveEnrollmentResultAsync(profile: VoiceProfile, cb?: (e: VoiceProfileEnrollmentResult) => void, err?: (e: string) => void): void {
```

Comment:
similar question to above, but the opposite.. When do we signal "Async" on function names? Is it indicated by the signature or by the calls occurring inside it?

Reply:
In this case, parity with Native API, which has a RetrieveEnrollmentResultAsync method on VoiceProfileClient. Unfortunately our "...Async" named APIs typically are not JS async, as they only offer parameterized callbacks. Currently the plan is to make the breaking change to async methods for v2.0.